lib/iris/fileformats/grib/message.py

```diff
@@ -33,17 +33,31 @@
 from iris.exceptions import TranslationError
 
 
-class _GribMessage(object):
+class _OpenFileRef(object):
     """
-    Lightweight GRIB message wrapper, containing **only** the coded keys and
-    data attribute of the input GRIB message.
+    A reference to an open file that ensures that the file is closed
+    when the object is garbage collected.
+    """
+    def __init__(self, open_file):
+        self.open_file = open_file
+
+    def __del__(self):
+        if not self.open_file.closed:
+            self.open_file.close()
+
+
+class GribMessage(object):
+    """
+    An in-memory representation of a GribMessage, providing
+    access to the :meth:`GribMessage.data` payload and the metadata
```

Comment:
These display as "GribMessage.data()" and "GribMessage.sections()".
You can avoid the noise of repeating the module path, by using the form :meth:`~GribMessage.data`  (i.e. add a tilde)